### PR TITLE
kibana 7.9.0

### PR DESCRIPTION
**Pull request by chenrui333**
Created with `brew bump-formula-pr`.

### CODE REVIEW

File: Formula/kibana.rb [hunk 1]
```diff
@@ -2,8 +2,8 @@ class Kibana < Formula
   desc "Analytics and search dashboard for Elasticsearch"
   homepage "https://www.elastic.co/products/kibana"
```
Comment (bayandin) [inline Formula/kibana.rb:3]: Let's try the same as https://github.com/Homebrew/homebrew-core/pull/59860#discussion_r472946157

```suggestion
  homepage "https://www.elastic.co/kibana"
```

**Comment by chenrui333** [comment]
```
==> node scripts/build --oss --release --skip-os-packages --skip-archives
internal/modules/cjs/loader.js:638
    throw err;
    ^

Error: Cannot find module '../../../../x-pack/plugins/reporting/server/browsers/install'
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:636:15)
    at Module.Hook._require.Module.require (/private/tmp/kibana-20200819-52238-16qum9g/node_modules/require-in-the-middle/index.js:51:29)
    at require (internal/modules/cjs/helpers.js:25:18)
```

**Comment by bayandin** [comment]
We explicitly remove `x-pack` directory: https://github.com/Homebrew/homebrew-core/blob/8d5adc5dae77b2e5f9028d80c093d39b81847096/Formula/kibana.rb#L21-L23 

The only place where it is used in `kibana` it's https://github.com/elastic/kibana/blob/v7.9.0/src/dev/build/tasks/install_chromium.js#L23 
and it actually doesn't do anything for OSS builds:
```js
export const InstallChromium = {
    ...
    if (build.isOss()) {
      return;
    } else {
      ...
    }
```

So we can workarounded something like:
```ruby
    inreplace "src/dev/build/tasks/index.ts", "export { InstallChromium } from './install_chromium';", ""
    inreplace "src/dev/build/build_distributables.ts", "await run(Tasks.InstallChromium);", ""
```

**Comment by bayandin** [comment]
```
==> Testing kibana
/usr/bin/sandbox-exec -f /private/tmp/homebrew20200826-37483-fkerab.sb ruby -W0 -I $LOAD_PATH -- /usr/local/Homebrew/Library/Homebrew/test.rb /usr/local/Homebrew/Library/Taps/homebrew/homebrew-core/Formula/kibana.rb --verbose
Configuring logger failed: { Error: EISDIR: illegal operation on a directory, read
    at Object.readSync (fs.js:498:3)
    at tryReadSync (fs.js:332:20)
    at readFileSync (fs.js:369:19)
    at Keystore.load (/usr/local/Cellar/kibana/7.9.0/src/legacy/server/keystore/keystore.js:70:45)
    at new Keystore (/usr/local/Cellar/kibana/7.9.0/src/legacy/server/keystore/keystore.js:29:10)
    at readKeystore (/usr/local/Cellar/kibana/7.9.0/src/cli/serve/read_keystore.js:33:20)
    at applyConfigOverrides (/usr/local/Cellar/kibana/7.9.0/src/cli/serve/serve.js:164:41)
    at applyConfigOverrides (/usr/local/Cellar/kibana/7.9.0/src/cli/serve/serve.js:223:42)
    at MapSubscriber.RawConfigService.config$.rawConfigFromFile$.pipe.rawConfig [as project] (/usr/local/Cellar/kibana/7.9.0/src/core/server/config/raw_config_service.js:39:16)
    at MapSubscriber._next (/usr/local/Cellar/kibana/7.9.0/node_modules/rxjs/internal/operators/map.js:49:35)
    at MapSubscriber.Subscriber.next (/usr/local/Cellar/kibana/7.9.0/node_modules/rxjs/internal/Subscriber.js:66:18)
    at ReplaySubject._subscribe (/usr/local/Cellar/kibana/7.9.0/node_modules/rxjs/internal/ReplaySubject.js:76:28)
    at ReplaySubject.Observable._trySubscribe (/usr/local/Cellar/kibana/7.9.0/node_modules/rxjs/internal/Observable.js:44:25)
    at ReplaySubject.Subject._trySubscribe (/usr/local/Cellar/kibana/7.9.0/node_modules/rxjs/internal/Subject.js:102:51)
    at ReplaySubject.Observable.subscribe (/usr/local/Cellar/kibana/7.9.0/node_modules/rxjs/internal/Observable.js:30:22)
    at MapOperator.call (/usr/local/Cellar/kibana/7.9.0/node_modules/rxjs/internal/operators/map.js:32:23) errno: -21, syscall: 'read', code: 'EISDIR' }

 FATAL  Error: EISDIR: illegal operation on a directory, read
```

it looks like this caused by https://www.elastic.co/guide/en/kibana/master/_kibana_keystore_has_moved_from_the_data_folder_to_the_config_folder.html#_kibana_keystore_has_moved_from_the_data_folder_to_the_config_folder

**Pull request closed without merging**